app/presentation/RoomItem/LastMessage.js

```diff
@@ -26,7 +26,8 @@ const formatMsg = ({
 	const isLastMessageSentByMe = lastMessage.u.username === username;
 
 	if (!lastMessage.msg && lastMessage.attachments && Object.keys(lastMessage.attachments).length) {
-		const user = isLastMessageSentByMe ? I18n.t('You') : lastMessage.u.username;
+		const name = useRealName ? lastMessage.u.name : lastMessage.u.username;
```

Comment:
Right below this, we have the same rule https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2897/files#diff-477299f3632d70ed68c5a4dedd1c1432968769cc1d582fde95cc395682242a3eR43

Can you just move your declaration above on this scope and use it there as well, so we keep it DRY?

Reply:
Sure. Will do :)